src/MongoDB.EntityFrameworkCore/Serializers/SerializationHelper.cs

```diff
@@ -40,61 +40,58 @@ public static T GetElementValue<T>(BsonDocument document, string elementName)
         return ReadElementValue<T>(document, serializationInfo);
     }
 
-    public static void WriteProperties(IBsonWriter writer, IUpdateEntry entry, Func<IProperty, bool>? propertyFilter = null)
+    internal static void WriteKeyProperties(IBsonWriter writer, IUpdateEntry entry)
```

Comment:
Why do we need `internal` here (and below)? the entire class is internal.

Reply:
Just to clarify we don't expect it to be called externally. 

The public two are the ones we bind to via reflection.